src/file-manager/fm-properties-window.c

```diff
@@ -564,6 +565,16 @@ create_image_widget (FMPropertiesWindow *window,
 				  G_CALLBACK (fm_properties_window_drag_data_received), NULL);
 		g_signal_connect (button, "clicked",
 				  G_CALLBACK (select_image_button_callback), window);
+
+		AtkObject * button_acc = gtk_widget_get_accessible (GTK_WIDGET (button));
+		gchar *icon_msg;
```

Comment:
The variable declaration should be at the top, just before `button = gtk_button_new ();`

Reply:
Done.